internal/pack/pack.go

```diff
@@ -315,3 +315,12 @@ func List(k *crypto.Key, rd io.ReaderAt, size int64) (entries []restic.Blob, err
 
 	return entries, nil
 }
+
+// PackedSizeOfBlob returns the size a blob actually uses when saved in a pack
+func PackedSizeOfBlob(blobLength uint) uint {
+	return blobLength + entrySize
+}
+
+func HeaderSize() int {
```

Comment:
This can be made a constant, if headerLengthSize is also made a constant; it's always 4. (Same goes for entrySize, but that's unrelated to this PR).

Reply:
changed.